addon/utils/domain-tweaks.js

```diff
@@ -0,0 +1,37 @@
+import { A } from '@ember/array';
+
+/**
+ * function to add custom ticks to the chart.
+ * @param {*} chart is the chart being referred - column-chart, barchart.
+ * @param {*} domain the un-padded domain.
+ * @param {*} otherTicks also add other ticks along with domain ticks because existing ticks are overwritten.
+ */
+export function addDomainTicks(chart, domain, otherTicks = undefined) {
```

Comment:
`undefined` is already the default value for params that are not passed in. Could do `otherTicks = []` instead to avoid the assignment on line 25.